docs/current/sdk/nodejs/snippets/private-repositories/clone.ts

```diff
@@ -0,0 +1,29 @@
+import Client, { connect } from "@dagger.io/dagger"
+
+// initialize Dagger client
+connect(async (client: Client) => {
+  // Collect value of SSH_AUTH_SOCK env var, to retrieve authentication socket path
+  const sshAuthSockPath = await client
```

Comment:
we're doing a different approach here from the [go example](https://github.com/dagger/dagger/pull/4251/files#diff-07752c09f8d3dd9036b0393d27fd1007342482e7cddafb15873494acf80371efR20). We don't need to call `client.host` here since we can get the value with `process.env.SSH_AUTH_SOCK`